README.md

```diff
@@ -229,15 +238,14 @@ A collection of awesome penetration testing resources
 * [Linux Exploit Suggester](https://github.com/PenturaLabs/Linux_Exploit_Suggester) - Linux Exploit Suggester; based on operating system release number.
 
 #### DDoS Tools
-* [LOIC](https://github.com/NewEraCracker/LOIC/) - An open source network stress tool for Windows
-* [JS LOIC](http://metacortexsecurity.com/tools/anon/LOIC/LOICv1.html) - JavaScript in-browser version of LOIC
```

Comment:
@tarrenj My opinion is that the tools should not be just viewed as tools to use but also as a source of knowledge.. I would still like to see LOIC on the list.. Maybe we can make a separate issue or separate and have community vote for it.. Can you put these back in this pull request and then create separate one to remove these?